mauth_client/utils.py

```diff
@@ -30,5 +30,5 @@ def decode(byte_string: bytes) -> str:
     try:
         return byte_string.decode("utf-8")
     except UnicodeDecodeError:
-        encoding = cchardet.detect(byte_string)["encoding"]
+        encoding = charset_normalizer.detect(byte_string)["encoding"]
```

Comment:
u may want to update my old comment on L28 which is specifically mentioning cchardet